solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -758,19 +758,19 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
 
         // avoid updating the rate more than once per block
         if (referenceRateUpdateTime == currentTime) {
-            return (false, Fraction({ n: referenceRateN, d: referenceRateD }));
+            return (false, referenceRate);
         }
 
         // get and store the effective rate between the reserves
         Fraction memory newRate = _effectiveTokensRate();
 
         // if the rate has changed, update it and rebalance the pool
-        if (newRate.n == referenceRateN && newRate.d == referenceRateD) {
+        Fraction memory ref = referenceRate;

```

Comment:
This (`Fraction memory ref = referenceRate`) is just for saving gas, right?

Reply:
Yeah